pkg/vm/manager.go

```diff
@@ -524,6 +525,13 @@ func (m *Module) Run(vm pkg.VM) (pkg.MachineInfo, error) {
 		return pkg.MachineInfo{}, m.withLogs(m.logsPath(vm.Name), err)
 	}
 
+	// clean up host keys before boot
+	command := exec.Command("rm", "-r", "/etc/ssh/ssh_host_*")
```

Comment:
Not that you should but if you really really really wants to use a `wildcard expansion` then what you need to do is `exec.Command("bash", "-c", "rm whatever*")` to make sure the bash expands the * before.

Which is ALSO not needed because go provide you with https://pkg.go.dev/path/filepath#Glob function